SharpAdbClient/SharpAdbClient.csproj

```diff
@@ -6,49 +6,33 @@
     <Title>.NET client for adb, Android Debug Bridge (SharpAdbClient)</Title>
     <VersionPrefix>2.2.0</VersionPrefix>
     <Authors>The Android Open Source Project, Ryan Conrad, Quamotion</Authors>
-    <TargetFrameworks>netstandard1.3;net451;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net451</TargetFrameworks>
     <AssemblyName>SharpAdbClient</AssemblyName>
     <AssemblyOriginatorKeyFile>SharpAdbClient.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>SharpAdbClient</PackageId>
-    <PackageTags>.net45;android;adb;netcore</PackageTags>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/quamotion/madb/master/LICENSE</PackageLicenseUrl>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>http://github.com/quamotion/madb</RepositoryUrl>
-    <PackageProjectUrl>http://github.com/quamotion/madb</PackageProjectUrl>

```

Comment:
Is this removal intended? Will it still show the link to the repository on nuget.org for example?

Perhaps `PublishRepositoryUrl` (which is still in this after the change) does take care of this, though I can't find any documentation that it does.

Reply:
Yes, `PublishRepositoryUrl` is a property of SourceLink, which will automatically add these URLs based on the information in the Git repository.

One of the advantages is that the repository URL is updated automatically for forked repositories, for example.

Reply:
That's great, thanks for explaining (and the work on this project in general)!